### PR TITLE
010 publications page

### PR DESCRIPTION
**Pull request by arielfenster**
Added comments to my files.
Front:
• Created PublicationsPage component. Big component
• Created a common resources directory (same level as routes) for shared methods between pages.

Back:
• Created a publications express middleware, exposed it to the server.js file and managed to call it. 
• Creating publications-data.json file

### CODE REVIEW

File: src/server/data/index-data.json [hunk 1]
```diff
@@ -1,51 +1 @@
...
-     },
-     {
-        "name":"Name 3",
-        "description":"Short Description Short Description Short Description ",
-        "link":{"info": "view project", "type": 1, "link": "aaa"}
-     }
-  ]
-}
\ No newline at end of file
+{"biography":"44","name":"444","age":"44","featuredPublications":[{"name":"65","description":"6","link":"66","type":"6"},{"name":"61","description":"2","link":"13","type":"21"}],"currentProjects":[{"name":"ok","description":"ok","link":"ok"}]}
```
Comment (teddy4445) [inline src/server/data/index-data.json:1]: I am not sure this is the right format...

File: src/server/data/publications-data.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[{"name":"321","description":"321","year":"651","authors":"321","topic":"321","type":"321","publisher":"321","publication status":"54","fileLinks":[{"info":"66","type":"44","link":"2"},{"info":"3","type":"1","link":"2"}]},{"name":"l","description":"k","year":"k","authors":"k","topic":"j","type":"kn","publisher":"kn","publication status":"j","fileLinks":[{"info":"8","type":"jh","link":"uh"}]}]
```
Comment (teddy4445) [inline src/server/data/publications-data.json:1]: I am not sure this is the right format

**Review by teddy4445 — APPROVED** [review]
Checkout the json file's format... everything else looks great

**Comment by teddy4445** [comment]
Anything else I can help with this PR?

File: src/server/data/index-data.json (hunk 1 above)
Comment (arielfenster) [reply src/server/data/index-data.json:1]: that was just a demo. i'll adjust the fields

File: src/server/data/publications-data.json (hunk 2 above)
Comment (arielfenster) [reply src/server/data/publications-data.json:1]: it seems to match the academic-publications.json file, except for the publication-status field that I need to fix. am I missing something?

**Pull request closed without merging**